src/stringClass.js

```diff
@@ -1,33 +1,165 @@
 const stringClass = {
+  /**
+   * This method takes a string and tests if it contains vowels
+   * @param {String} this
+   * @return {Boolean} true or false
+   */
   hasVowels() {
     return /[aeiou]/.test(this);
   },
 
-  toUpper() { },
+/**
+   * This method takes a string and returns it in uppercase
+   * @param {String} this
+   * @return {String} this
+   */
+  toUpper() {
+    return this.replace(/[a-z]/g, caps =>
+      String.fromCharCode(caps.charCodeAt(0) - 32)
+    );
+  },
 
-  toLower() { },
+/**
+   * This method takes a string and returns it in lowercase
+   * @param {String} this
+   * @return {String} this
+   */
+  toLower() {
+    return this.replace(/[A-Z]/g, lower =>
+      String.fromCharCode(lower.charCodeAt(0) + 32)
+    );
+  },
 
-  ucFirst() { },
+/**
+   * This method takes a string and returns its first letter in uppercase
+   * @param {String} this
+   * @return {String} this
+   */
+  ucFirst() {
+    return this.replace(this[0], this[0].toUpper());
+  },
 
-  isQuestion() { },
+/**
+   * This method takes a string and checks if it's a question
+   * @param {String} this
+   * @return {Boolean} true or false
+   */
+  isQuestion() {
+    return /\?$/.test(this);
+  },
 
-  words() { },
+/**
+   * This method takes a string and check if it has multiple words
+   * @param {String} this
+   * @return {Array} array
+   */
+  words() {
+    return this.replace(/[^\w\s]/g, '').split(/\s+/);
+  },
+/**
+   * This method takes a string and returns the number of words it contains
+   * @param {String} this
+   * @return {Integer} number
+   */
+  wordCount() {
+    return this.words().length;
+  },
 
-  wordCount() { },
+/**
+   * This method takes a string in float type and returns it in currency format
+   * @param {String} this
+   * @return {String} this
+   */
+  toCurrency() {
+    const digits = this.split(/\./);
+    digits[1] = digits[1] || '00';
+    digits[0] = digits[0].split('').reverse().join('')
+      .replace(/\d{3}/, value => `${value},`)
+      .split('')
+      .reverse()
+      .join('');
+    return `${digits[0]}.${digits[1]}`;
+  },
 
-  toCurrency() { },
+/**
+   * This method takes a string in currency format and converts to float type
+   * @param {String} this
+   * @return {String} this
+   */
+  fromCurrency() {
+    return this.replace(/,/g, '');
+  },
```

Comment:
- You may want to take care of insignificant zeroes here. e.g `'123,456.00'.fromCurrency()` should return `'123456'`